packages/store-sync/src/dozer/decodeDozerField.ts

```diff
@@ -0,0 +1,24 @@
+import { AbiType } from "@latticexyz/config";
+import {
+  ArrayAbiType,
+  SchemaAbiTypeToPrimitiveType,
+  arrayToStaticAbiType,
+  schemaAbiTypeToDefaultValue,
+} from "@latticexyz/schema-type/internal";
+
+export function decodeDozerField<abiType extends AbiType>(
```

Comment:
what about `decodeField` or `decodeColumnValue`? (we're in the dozer export, so dozer in the name seems duplicative)